lib/elixir/lib/enum.ex

```diff
@@ -1341,8 +1341,8 @@ defmodule Enum do
   If multiple elements are considered maximal, the first one that was found
   is returned.
 
-  Calls the provided `empty_fallback` function and returns its value if
-  `enumerable` is empty. The default `empty_fallback` raises `Enum.EmptyError`.
+  Calls the provided `empty_fallback/0` function and returns its value if
```

Comment:
The documentation should say argument `fun` is a  function that expects arguments x, y, z and must return w.